src/main/java/TaskManager.java

```diff
@@ -0,0 +1,121 @@
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class TaskManager {
+    private final List<Task> recordedTasks;
+    private int taskCount = 0;
+
+    TaskManager() {
+        this.recordedTasks = new ArrayList<Task>();
+    }
+
+    /**
+     * Creates a new task.
+     * 
+     * @param command a string from user input
+     * @param args an array of strings from user inputs
+     */
+    public final void addTask(String command, String[] args) {
+        if (args.length == 0) {
+            ConversationManager
+                    .printErrorResponse(List.of("no such command or insufficient arguments"));
```

Comment:
Good job splitting a long statement into two lines and ensuring that the line length is not more than 120 char